app/(storyblok)/[...slug]/page.tsx

```diff
@@ -1,40 +1,22 @@
-import { Metadata } from 'next';
-import { getStoryblokApi, ISbStoriesParams, StoryblokClient } from '@storyblok/react/rsc';
-import StoryblokStory from '@storyblok/react/story';
+import { Metadata, ResolvingMetadata } from 'next';
+import {
+ getStoryblokApi, ISbStoriesParams, StoryblokClient, StoryblokStory,
+} from '@storyblok/react/rsc';
 import { resolveRelations } from '@/utilities/resolveRelations';
 import { getPageMetadata } from '@/utilities/getPageMetadata';
+import { notFound } from 'next/navigation';
```

Comment:
This little function triggers the 404 (not-found.tsx) page instead of throwing an error which hits an error boundary and triggers error.tsx.

Reply:
Noice